boards: nordic: nrf5340dk/nrf9160dk: Use sysbuild for ns twister

nordicjm opened the pull request:
Enables forcing use of sysbuild to build nrf5340dk/nrf9160dk non-secure board targets when using twister

Also fixes #84233

nordicjm committed:
boards: nordic: nrf5340dk/nrf9160dk: Use sysbuild for ns twister

Enables forcing use of sysbuild to build nrf5340dk/nrf9160dk
non-secure board targets when using twister

Signed-off-by: Jamie McCrae <jamie.mccrae@nordicsemi.no>

nordicjm commented:
@tomi-font there's failures here with the ns board targets in mbedtls, can you take a look?

tomi-font commented:
Try enabling test entropy for the failing tests (`CONFIG_TEST_RANDOM_GENERATOR` and maybe also `CONFIG_TIMER_RANDOM_GENERATOR`).

kartben commented:
@nordicjm rebasing should do the trick I believe, this issue has been fixed in main

nordicjm committed:
tests: kernel: pipe: deprecated: Fix test wrongly passing paramater

Fix these tests so that the parameters are correctly passed to
other parts of the build

Signed-off-by: Jamie McCrae <jamie.mccrae@nordicsemi.no>

nordicjm commented:
please review @anangl @masz-nordic
